web/ui/react-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prometheus-io/app",
-  "version": "0.38.0",
+  "version": "0.39.0-rc.0",
```

Comment:
weirdly the script `bump-version` should have updated as well the dependencies in this package.json :O